controllers/upgrade.go

```diff
@@ -418,7 +433,7 @@ func (r *RollingUpgradeContext) ReplaceNodeBatch(batch []*autoscaling.Instance)
 	return true, nil
 }
 
-func (r *RollingUpgradeContext) SelectTargets(scalingGroup *autoscaling.Group) []*autoscaling.Instance {
+func (r *RollingUpgradeContext) SelectTargets(scalingGroup *autoscaling.Group, failedDrainInstances []string) []*autoscaling.Instance {
```

Comment:
```suggestion
func (r *RollingUpgradeContext) SelectTargets(scalingGroup *autoscaling.Group, excludedInstances []string) []*autoscaling.Instance {
```

Reply:
fixed in [1fab0cc](https://github.com/keikoproj/upgrade-manager/pull/394/commits/1fab0cc975e4f7af8b35f4e89a4244467683f4bc)